.travis.yml

```diff
@@ -1,48 +1,57 @@
-language: python
 dist: xenial
-sudo: false
+language: python
+cache: pip
 python:
-- '3.5'
-- '3.6'
-- '3.7'
-- '3.8'
+  - 3.6
```

Comment:
Any specific reason why have we removed python 3.5 support?

Reply:
@nasirhjafri this is is just a practice I have adopted from some other packages I maintain.

Python 3.5 is not supported by newer Django versions and it simplifies things to offer support on 3.6+ feature wise. 3.6 has a lot of forwards compatible features. Python 3.5 is also dropping in usage numbers as most distributions offer 3.6+ as the stock distribution.

Reply:
Django 2.2  is still supporting python 3.5 since we are supporting Django 2.2 let's also support Python 3.5 too.

Reply:
OK, I'll add it to the matrix!